docs/swagger.json

```diff
@@ -1557,6 +1533,16 @@
             },
             "TaskResult": {
                 "properties": {
+                    "message": {
+                        "type": "string",
+                        "title": "Message",
+                        "description": "Result detail description"
+                    },
+                    "error": {
+                        "type": "string",
+                        "title": "Error",
+                        "description": "If the task status result is `False` shows an error message"
```

Comment:
Is this API  user facing documentation? They don't know about task status, right?